README.md

```diff
@@ -6,7 +6,7 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 [![CLA assistant](https://cla-assistant.io/readme/badge/ibm/cloud-databases-go-sdk)](https://cla-assistant.io/ibm/cloud-databases-go-sdk)
 
-# IBM Cloud Databases Go SDK 0.1.0
+# IBM Cloud Databases Go SDK 0.2.0
```

Comment:
This should not be done manually.  It will be changed by semantic-release when the PR is merged into main.
```suggestion
# IBM Cloud Databases Go SDK 0.1.0
```